src/main/java/com/TableTalk/Enterprise/dao/GameDAOStub.java

```diff
@@ -40,7 +40,6 @@ public GameCollection fetchGamesByName(String inputtedName) throws IOException {
         Map<String, String> filter = new HashMap<>();
         filter.put("name", inputtedName);
         filter.put("client_id", CLIENT_ID);
-        System.out.println(filter);
```

Comment:
System.out.println(filter); i was unsure as to why this was here, i assume it was to verify the data in filter but is it still needed?

Reply:
Use proper logging instead of System.out.println().
- Loggins has the concept of priority (error, debug, warn, etc.), which System.out.println doesn't have.
- No one will see System.out.println(), as it goes to console. 
- Log messages can be aggregated by several commercial tools.  Not so easy with System.out.println.